upload/Validate.php

```diff
@@ -1,46 +1,46 @@
 <?php
+
 namespace upload;
 
 class Validate extends FileInfo implements ValidateInterface
 {
+    public $Errors = [];
+
+    public function __construct($FilePath)
+    {
+        if (is_uploaded_file($FilePath)) {
+            parent::__construct($FilePath);
+        } else {
+            $this->Errors[] = 'Files Was Not Uploaded';
+        }
+    }
 
-  public $Errors = array();
+    public function Size($AllowedSize)
+    {
+        if ($this->getSize() > $AllowedSize) {
+            $this->Errors[] = 'File size Not Allowed : '.$this->getName();
+        }
 
-  function __construct($FilePath)
-  {
-    if (is_uploaded_file($FilePath)) {
-      parent::__construct($FilePath);
-    } else {
-      $this->Errors[] = "Files Was Not Uploaded";
+        return $this;
     }
-  }
 
-  public function Size($AllowedSize)
-  {
-    if($this->getSize() > $AllowedSize) {
-      $this->Errors[] = "File size Not Allowed : ".$this->getName();
+    public function Extension($AllowedExtension)
+    {
+        $AllowedExtension = explode(',', $AllowedExtension);
+        $AllowedExtension = array_map('strtolower', $AllowedExtension);
+        if (!in_array($this->getExtension(), $AllowedExtension) and $this->getExtension()) {
+            $this->Errors[] = 'Extension Not Allowed : '.$this->getExtension();
+        }
+
+        return $this;
     }
-    return $this;
-  }
-
-  public function Extension($AllowedExtension)
-  {
-    $AllowedExtension = explode(",", $AllowedExtension);
-    $AllowedExtension = array_map('strtolower', $AllowedExtension);
-      if (!in_array($this->getExtension(), $AllowedExtension) and $this->getExtension()) {
-        $this->Errors[] = "Extension Not Allowed : ".$this->getExtension();
-      }
-    return $this;
-  }
-    
-  public function Exist()
-  {
-      if (!$this->getName()) {
-        $this->Errors[] = "File was Not selected";
-      }
-    return $this;
-  }
 
-}
+    public function Exist()
```

Comment:
__[code_sniffer]__
* Method name "Validate::Exist" is not in camel caps format

<!-- Comment by [SideCI](https://sideci.com) -->